README.md

```diff
@@ -1,4 +1,151 @@
-# Awesome NEAR [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
+# Awesome NEAR
 
 A curated list of awesome NEAR frameworks, libraries, software and resources.
 
+## Contacts
+
+* [Website](https://near.org)
+* [Discord](https://near.chat)
+
+## Documentation
+
+* [NEAR Docs](https://docs.near.org/)
+* [NEAR Blog](https://near.org/blog/)
+* [NEAR Wiki](https://wiki.near.org/)
+* [NEAR Specs](https://nomicon.io/)
+
+## Libraries / Frameworks
+
+Smart Contract SDK
+
+* [near-sdk-rs](https://github.com/near/near-sdk-rs)
+* [near-sdk-js](https://github.com/near/near-sdk-js)
+* [near-sdk-as](https://github.com/near/near-sdk-as)
+
+Frontend Libraries
+
+* [near-api-js](https://github.com/near/near-api-js)
+* [near-api-go](https://github.com/textileio/near-api-go)
```

Comment:
btw, we have another one from Aurora: https://github.com/aurora-is-near/near-api-go